frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -99,6 +104,7 @@ export class TransformTopologyData {
   private deploymentKindMap = {
     deployments: { dcKind: 'Deployment', rcKind: 'ReplicaSet', rController: 'replicasets' },
     daemonSets: { dcKind: 'DaemonSet', rcKind: 'ReplicaSet', rController: 'replicasets' },
+    statefulSets: { dcKind: 'StatefulSet', rcKind: 'ReplicaSet', rController: 'replicasets' },
```

Comment:
Replace all the `dcKind` property's value with the appropriate property from the newly created enum Kind

Reply:
fixed in https://github.com/openshift/console/pull/2162/commits/6e1b2bfc91fa4abfbf7f7a124df61c4884702c89